src/magellan_core/launch/passive.launch

```diff
@@ -2,7 +2,7 @@
     <node pkg="tf2_ros" type="static_transform_publisher" name="imu_link_broadcaster" args="0 0 0 1.57 0 0 base_link imu" />
 
     <node pkg="magellan_core" type="viz_turning_radius.py" name="viz_turning_radius" />
-
+  
```

Comment:
why are there whitespace changes in here? can we clean this up

Reply:
fixed